### PR TITLE
Responsive-cake-website-wireframe/tayfuntayfun

### PR DESCRIPTION
**Pull request by tayfuntayfun**

### CODE REVIEW

**Comment by tayfuntayfun** [comment]
This is the mobile view to be converted onto the desktop . Keeping to work on this @media tomorrow.

File: index.html [hunk 1]
```diff
@@ -6,10 +6,70 @@
...
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
+    <link rel="stylesheet" href="style.css">
 <body>
+    <header class="mainheader">
+        <div class="topheader">
+            <h3 class="topheader1">LOGO</h3>
```
Comment (LucyMac) [inline index.html]: Logo will probably need to be an `img` tag not an h3

File: index.html [hunk 2]
```diff
@@ -6,10 +6,70 @@
...
+    </section>
+    <section class="cake-wrapper-row">
+        <div class="cake-wrapper-row-part">
+            <figure class="wrapperimage">
+                <div class="cakeswrapper">CAKE1</div>
+                <div class="cakesdetails">Lorem ipsum dolor sit amet, consectetur adipiscing elit</div> 
+            </figure>
+            <figure class="wrapperimage">
+                <div class="cakeswrapper">CAKE2</div>
+                <div class="cakesdetails">Lorem ipsum dolor sit amet, consectetur adipiscing elit</div>
```
Comment (LucyMac) [inline index.html]: You could use `<figcaption>` for this since you have nested it inside a `figure`

File: style.css [hunk 3]
```diff
@@ -1 +1,268 @@
 /*  Add your styling here */
+body{
```
Comment (LucyMac) [inline style.css]: You have 2 files with the same name `style.css`? If you want to split your CSS, make sure you use different file names. If you only need one, then delete the other one 🙂

File: index.html (hunk 1 above)
Comment (tayfuntayfun) [reply index.html]: REplaced with img tag

File: style.css (hunk 3 above)
Comment (tayfuntayfun) [reply style.css]: Deleted the redundant one :)

File: index.html (hunk 2 above)
Comment (tayfuntayfun) [reply index.html]: Changed to figcaption. Learned another thing, thanks Lucy. You are a star.

**Pull request closed without merging**